backends/opentsdb/README.md

```diff
@@ -23,7 +23,7 @@ change the `destination = localhost:4242` line accordingly.
 
 As of [v1.16.0](https://github.com/netdata/netdata/releases/tag/v1.16.0), Netdata can send metrics to OpenTSDB using
 TLS/SSL. Unfortunately, OpenTDSB does not support encrypted connections, so you will have to configure a reverse proxy
-to enable HTTPS communication between Netdata and OpenTSBD. You can set up a reverse proxy with
+to enable HTTPS communication between Netdata and OpenTSDB. You can set up a reverse proxy with
```

Comment:
This seems like a correct change in context (the changes were done in a fairly automated manner)